core/src/com/dragons/game/model/GameWorld/GameWorld.java

```diff
@@ -0,0 +1,84 @@
+package com.dragons.game.model.GameWorld;
+
+
+import com.badlogic.gdx.math.Vector2;
+import com.badlogic.gdx.physics.box2d.World;
+import com.dragons.game.model.ISubject;
+import com.dragons.game.model.bomb.Bomb;
+import com.dragons.game.model.player.Player;
+
+import java.util.ArrayList;
+
+/**
+ * The GameWorld class instantiates the world in which a single game will be played. The
+ * class itself then works as a container for our game where every time-step is simulated with
+ * the proper actions and interactions performed on and in between objects. After an object is
+ * created, it therefore has to be added to the game-world.
+ *
+ * @author Eldar Sandanger
+ * */
+
+public class GameWorld {
```

Comment:
Denne klassen er grei å se over, flott hvis dere skjønner det, men ikke krise hvis ikke. Det viktigste er at når vi lager et objekt som skal plasseres på brettet, så skal det legges inn her gjennom addObject. For eksempel: når du oppretter en block oppretter du den som du vanligvis ville gjort, så skriver du addObject(*blokken*, *statisk true/false*)